lldb/test/Shell/Swift/function-name-backtrace.test

```diff
@@ -0,0 +1,52 @@
+# XFAIL: target-windows
```

Comment:
I would've expected this patch to be NFC, and hence not require new tests. Did we just never have coverage for swift backtraces in the test-suite?